### PR TITLE
Hangman game added

### LINKED ISSUE #937: Web Development: Hangman Game
### Is your feature request related to a problem? Please describe.

Yes

### Describe the solution you'd like.

I want to create a fun and educational Hangman game.

## Tech Stack Used

- HTML
- CSS
- Javascript

### Describe alternatives you've considered.

NA

### Add any other context or screenshots about the feature request here.

image

### PR DESCRIPTION
**Pull request by rittikadeb**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes

### Description

Added a fun and educational Hangman Game.

Fixes #937 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

No

### CODE REVIEW

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@rittikadeb Changes the below things: 

1.  Deploy the project and add the deploy link in `readme.md` file .
2.  Add the project name and count in the web development `readme.md` file.

**Comment by rittikadeb** [comment]
This project is under Haktoberfest 2023.

File: Web Development/README.md [hunk 1]
```diff
@@ -21,22 +21,22 @@
...
-| 67. | [Food Menu Cart](https://github.com/Kushal997-das/Project-Guidance/tree/main/Web%20Development/Basic/Food%20Menu%20Cart) |  68. | [Weight Converter](https://github.com/Kushal997-das/Project-Guidance/tree/main/Web%20Development/Basic/Weight-Converter) | 69. | [Zomato Landing Page](https://github.com/Kushal997-das/Project-Guidance/tree/main/Web%20Development/Basic/Zomato-Landing-Page) | 
+| 22. | [Google Clone](https://github.com/SamarpanCoder2002/Project-Guidance/tree/main/Web%20Development/Basic/Google%20Clone) | 23. | [Google Homepage Clone](https://github.com/rajprem4214/Project-Guidance/tree/gclone/Web%20Development/Basic/Google%20Homepage%20Clone) | 24. | [Hangman Game](https://github.com/rittikadeb/Project-Guidance/tree/hangman-game/Web%20Development/Basic/Hangman%20Game) | 25. | [Instagram Login Page Clone](https://github.com/abhilipsasahoo03/Project-Guidance/tree/main/Web%20Development%2FBasic%2FInstagram%20Login%20Page%20Clone) |
```
Comment (Kushal997-das) [inline Web Development/README.md:24]: 25 number is missing , kindly check

**Comment by rittikadeb** [comment]
@Kushal997-das pls review it

**Review by Kushal997-das — APPROVED** [review]

**Pull request merged**